get-started/versioning-availability.md

```diff
@@ -7,208 +7,53 @@ products:
 description: Learn how Elastic handles versioning and feature availability in the docs. Find the product versions that are supported, how to read availability badges, and...
 ---
 
-# Versioning and availability in Elastic Docs
+# Versioning and availability
 
-Learn how Elastic Docs handles versioning, feature availability, and how to find the right documentation for your deployment type and product version. Find answers to common questions about the Elastic Stack versioning and confidently navigate our continuously updated documentation.
+Learn how Elastic products are versioned, the lifecycle of features, and how to find the right documentation for your deployment type and product version. Find answers to common questions about the versioning and confidently navigate our continuously updated documentation.
```

Comment:
Should we make this page a lot less about the docs and more about the product lifecycle?

If it's more about the docs, then we can merge it into the "how to use the documentation" page.